apps/ejabberd/src/mongoose_acc.erl

```diff
@@ -178,12 +183,35 @@ require([Key|Tail], Acc) ->
 require(Key, Acc) ->
     require([Key], Acc).
 
-%% @doc Remove all data we cached
-%% Personally I think we should consider a more flexible implementation of
-%% in-accumulator cache, so that attrs are not hardcoded here.
--spec flush(t()) -> t().
-flush(Acc) ->
-    remove(privacy_check, Acc).
+%% @doc Convert the acc before routing it out to another c2s process - remove everything except
+%% the very bare minimum (all caches, records etc), replace element with to_send
+-spec strip(t()) -> t().
+strip(Acc) ->
+    El = get(to_send, Acc),
+    Acc1 = from_element(El),
+%%    Ats = [name, type, attrs, from, from_jid, to, to_jid, ref, timestamp],
+    Ats = [from, from_jid, ref, timestamp],
```

Comment:
Does `Ats` stands for `Attributes`? If so I think more readable is `Attrs` or even whole `Attributes`